README.md

```diff
@@ -43,3 +43,54 @@ To run local server in production mode:
 ```sh
 pnpm ssdeploy run
 ```
+
+
+## HTTP API
```

Comment:
Let’s add it above the Development. I have a theory that most people will send PR without starting th server.